lib/datadog/tracing/contrib/active_support/cache/redis.rb

```diff
@@ -22,17 +22,29 @@ module Patcher
               # For Rails >= 5.2 w/o redis-activesupport...
               # ActiveSupport includes a Redis cache store internally, and does not require these overrides.
               # https://github.com/rails/rails/blob/master/activesupport/lib/active_support/cache/redis_cache_store.rb
-              def patch_redis?(meth)
+              def patch_redis_activesupport?(meth)
```

Comment:
I find the naming confusing because later we anyway patching something within `ActiveSupport`, WDYT of this name?

```suggestion
              def patch_redis_store?(meth)
```

It follows the idea to target specific store, and next method is a good example, you get what you've asked, patch either redis store or redis cache store.

Reply:
Done!